tests/NexusMods.Paths.Tests/SizeTests.cs

```diff
@@ -8,9 +8,6 @@ public void MathAndSize()
         var a = (Size)10L;
         var b = Size.FromLong(20L);
 
-        a.Should().BeLessThan(b);
-        b.Should().BeGreaterThan(a);
```

Comment:
Should these not work now?

Reply:
Fluent Assertions is weird, these methods didn't resolve properly due to generics. Not really an issue since the code below tests this behavior already.

Reply:
Ah, okay, I see.
The fact these checks were removed were why I originally asked.